### DISCUSSION TITLE
Pass data from your components to your layout

### DISCUSSION THREAD

**Discussion by rafael1delgado** [livewire/livewire#4715] [1. Help] [answered]
i all, how to pass a variable to extended layout in livewire render function. When I tried to pass a variable , it is not displaying in layout blade. Showing undefined variable error. What is the correct method to do that.

`return view('livewire.show-posts')->layout('layouts.base', ['title' => 'Show Posts'])`

the page title is not displayed as expected

**Comment by joshhanley** [comment, marked as answer]
This is a duplicate of #4704. Please don't open two discussions with the same thing. We will continue the discussion on #4704.
